@here/harp-mapview/lib/text/Placement.ts

```diff
@@ -672,6 +681,23 @@ function placePointLabelAtAnchor(
         // It might be changed if we would like to render text without icon (at border, etc.).
         return persistent ? PlacementResult.Rejected : PlacementResult.Invisible;
     }
+
+    if (measureText) {
+        // Bigger label margin is applied only for the first time it is placed and its visible
+        // with its normal bounds. This margin compensates the changes of label bounds due to
+        // distance scaling and it is designed to cover size changes due to camera panning.
+        // This ways label "reserves" enough space so it may "grow" without affecting other
+        // labels, no collisions occurs and thus limits labels flickering on pan.
+        tmpBox.getSize(tmpSize);
+        tmpSize.multiplyScalar(scale * (1 + newPointLabelTextMarginPercent));
```

Comment:
ok, so you went finally for applying the extra margin to the scaled bbox. it should be fine as soon as we fix how we compute the scaling factor so that it doesn't affect top down views.

Reply:
Yep, I have already tried it and seems to work fine, would you like me to add this change to this PR?